rfd/0074-sftp-support.md

```diff
@@ -0,0 +1,136 @@
+---
+authors: Andrew LeFevre (andrew.lefevre@goteleport.com)
+state: draft
+---
+
+# RFD 74 - SFTP Support
+
+## Required Approvers
+
+- Engineering: `@r0mant && @jakule`
+- Product: `@klizhentas || @xinding33`
+
+## What
+
+Add SFTP support to `tsh` and Teleport Node and Proxy services. This will
+not replace scp, both will be supported.
+
+## Why
+
+[OpenSSH 9.0](https://www.openssh.com/txt/release-9.0) changed `scp` to
+use the [SFTP](atatracker.ietf.org/doc/html/draft-ietf-secsh-filexfer-02)
+protocol by default instead of scp/rcp protocol. This is to
+combat many security issues present in the legacy scp/rcp protocol. Users
+running the latest OpenSSH version will currently find that `scp` and various
+tools that use it under the hood, such as Ansible and various JetBrain IDEs
+will not work through Teleport anymore. This can be fixed temporarily by passing
+a flag to `scp`, but that can be difficult when `scp` is being invoked indirectly
+by other tools.
+
+## Details
+
+[github.com/pkg/sftp](https://pkg.go.dev/github.com/pkg/sftp) will be leveraged 
```

Comment:
@r0mant please request audit of this library and the code with our security team and pentest team